dapr/actor/runtime/actor.py

```diff
@@ -49,16 +49,14 @@ async def _on_deactivate(self):
     def __init__(self, ctx: ActorRuntimeContext, actor_id: ActorId):
         self.id = actor_id
         self._runtime_ctx = ctx
-        self._timers: Dict[str, ActorTimerData] = {}
-        self._timers_lock = asyncio.Lock()
         self._state_manager: ActorStateManager = ActorStateManager(self)
 
     @property
     def runtime_ctx(self) -> ActorRuntimeContext:
         return self._runtime_ctx
 
     def __get_new_timer_name(self):
-        return f'{self.id}_Timer_{len(self._timers) + 1}'
+        return f'{self.id}_Timer_{uuid.uuid4()}'
```

Comment:
counter is actually not a good solution anymore because app can restart. So, I am using UUID to make sure it is unique (at least, highly probable).